src/main/java/com/solvd/carina/demo/gui/components/zebrunner/navigation/ZebRunnerNavigationMenuBase.java

```diff
@@ -0,0 +1,22 @@
+package com.solvd.carina.demo.gui.components.zebrunner.navigation;
+
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+
+public abstract class ZebRunnerNavigationMenuBase extends AbstractUIObject {
```

Comment:
I believe this class helps to encapsulate the common attributes and behavior of NavigationMenu class and also will help to easily add new functionalities to its subclasses. I think it would be beneficial if I am planning to do mobile automation in future. I don't have to start from the scratch instead can create a new subclass of NavigationMenuBase.